scripts/stale.py

```diff
@@ -37,19 +37,20 @@ def main():
     open_issues = repo.get_issues(state="open")
 
     for issue in open_issues:
+        print(issue)
```

Comment:
I'm leaving this print in in case we'd like to debug; it wasn't possible previously.